### DISCUSSION TITLE
Disable warning about configuration file not found

### DISCUSSION THREAD

**Discussion by sergejostir** [drakkan/sftpgo#269] [Ideas]
We can now set all configuration options with env vars without using our own config file (https://github.com/drakkan/sftpgo/commit/ebb18fa57d34136019fbf5dfa8d40540d89438c3). However, on every startup a warning is logged saying that the configuration file could not be found.

I guess there should be an option added to specify that sftpgo shouldn't go looking for a configuration file?

**Comment by drakkan** [comment]
Hi,

I think we don't need a new configuration option for this, please take a look [here](https://github.com/drakkan/sftpgo/commit/c3410a3d914ea0369c8bba92e44f9b736b9332fc) thank you!
